### DISCUSSION TITLE
Project Pythia Feedback

### DISCUSSION THREAD

**Discussion by jukent** [ProjectPythia/projectpythia.github.io#189] [Ideas]
During the OWG meeting we discussed a Project Pythia feedback form. Here is the rough [feedback google form](https://docs.google.com/forms/d/e/1FAIpQLSeVa1TC9xM-dk7qIE2e8bsgSrIP82yYDNw3wew3J46eREJa4w/viewform?usp=sf_link).

Everyone on the projectpythia@ucar.edu listserv should be able to edit it.

What questions do you think should be edited/added/removed? 
Where should we embed the form? On the portal or the foundations book? Both?

**Comment by dcamron** [comment]
To balance brevity with helpful-ness, other sentiments I see fit to consider evaluating:

 * I (will/do) use skills demonstrated here in my school/workplace
 * I learned something new
 * I was engaged by the materials (maybe more "interesting/well-presented" than "engaging" in this format)
 * I was challenged by the materials (maybe more "in-depth/complex" than "challenging" in this format)

Some of these sentiments might alright be nestled in the existing questions.

Will we get enough out of "yes/no" questions as opposed to scale-ranked (e.g. 1-5) questions? I like yes/no for the super brief presentation, and it's likely _enough_ for this sort of "drive-by" feedback. We could potentially even collapse it down into a single "check all that apply: I felt comfortably challenged, I completed the material, the material provides me with skills for my workplace, etc."

We also can provide a drop-down for choosing the material so that they're not having to freehand it, if that won't balloon into something too large. Unless we prefer to leave room for the respondents to specify an entire section versus a single notebook.

Where: I think for now this could just be a part of the foundations material, as that's the main material we have total control over and would respond to feedback directly on ourselves. We can include a link at the end of every notebook and content section with some sort of guiding text about providing feedback.

Thanks for drafting this and starting this discussion @jukent!

**Comment by jukent** [comment]
Okay then I should move this discussion to that repo
